ui/components/KustomizationDetail.tsx

```diff
@@ -54,27 +54,27 @@ export default function KustomizationDetail({ kustomization, name }: Props) {
           suspended={kustomization?.suspended}
         />
       </Flex>
-      <TabContent wide column align>
```

Comment:
I THINK this is what's causing this on my big screen 
![image](https://user-images.githubusercontent.com/65822698/162838621-dbedf519-a31f-4168-84aa-3af2d2ef3e31.png)
Pretty much every container for the tab stuff needs `wide` and `tall`

Reply:
Cannot reproduce:
![Screenshot from 2022-04-12 08-27-56](https://user-images.githubusercontent.com/2802257/162998607-fb618620-315c-4b91-b049-8e1a1102ee28.png)

I don't think the `tall` attribute exists on my branch?

Reply:
Once I rebased, I saw this. Fixed.